guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p><s>Finally t</s>The model is now ready for training<s>, and </s> <u>using </u>the <code>model.fit()</code> method. <s>is used which</s> The <code>model.fit()</code> is given the number of epochs for training and the estimated learning rate <s>selected based on</s> <u>generated </u>by <s>the</s> <code>lr_find()</code> <s>returned</s> in the previous step:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>